ansible/library/pulp_facts.py

```diff
@@ -23,19 +24,40 @@
 projects = [
     'crane',
     'pulp',
-    'pulp_deb',
-    'pulp_docker',
-    'pulp_openstack',
-    'pulp_ostree',
-    'pulp_puppet',
-    'pulp_python',
-    'pulp_rpm'
+    'pulp-deb',
+    'pulp-docker',
+    'pulp-openstack',
+    'pulp-ostree',
+    'pulp-puppet',
+    'pulp-python',
+    'pulp-rpm'
 ]
 rpm_dependency_list = []
 # This is run using sudo, but the code is checked out in the normal user directory.
 user_homedir = pwd.getpwuid(int(os.environ['SUDO_UID'])).pw_dir
+
+# Figure out the platform and checkout the packaging repository branch
+platform_dist = platform.linux_distribution()
+if platform_dist[0].lower() == 'fedora':
+    # Maybe we'll support other platforms for development, but right now Fedora is
+    # all we work with.
+    branch = 'f' + str(platform_dist[1])
+else:
+    raise OSError(str(platform_dist) + ' is not a supported development platform')
+
+
+packaging_repo = os.path.join(user_homedir, 'devel/packaging/')
```

Comment:
I'm on the fence about requiring the packaging repository to be checked out here. The alternative is to check it out in a temporary location and not rely on the user to clone it.